infra/docker/keycloak/Dockerfile

```diff
@@ -14,6 +14,8 @@ RUN mvn clean package
 
 FROM quay.io/keycloak/keycloak:25.0.2 AS builder
 
+ARG DASHBOARD_CLIENT_URL
```

Comment:
This env var is dynamic and should not be used at build time, only at launch time.